### ISSUE TITLE
No partition delivery error

### ISSUE THREAD

**Issue by bonanza123** [Androxyde/Flashtool#109]
I was trying to flash my Sony Xperia XA2 Ultra(H3213) but I get the
`No partition delivery`
error. I downloaded the image via XperiaFirm linked within the flashtool.

Please see the following full log
https://gist.github.com/bonanza123/616cdd98ee1e274567b9e604041b71bf
Is it safe to continue without parition delivery?
Is this an error on flashtool or XperiaFirm side?

**Comment by giosal** [comment]
Same here for Xperia XA2 (H3113)
No partition delivery error. Tried using last flashtool and xperifirm.
Tried download different region files.

**Comment by bonanza123** [comment]
@giosal I ended up using EMMA, worked flawlessy.

**Comment by giosal** [comment]
@bonanza123 can you send me the link in PM

**Comment by bonanza123** [comment]
@giosal https://developer.sony.com/develop/open-devices/get-started/flash-tool/download-flash-tool/

**Comment by giosal** [comment]
@bonanza123 thanks a lot, I'll give it a try

**Comment by giosal** [comment]
@bonanza123, for me, the problem is that I haven't unlocked bootloader and, honestly, I don't want to do that, as I'm just trying to flash official rom and not custom rom...
I've read already that this is possible wiithout unlocked bootloader

**Comment by bonanza123** [comment]
True, you can use EMMA only if you unlocked your bootloader. But I guess the same holds for flashtool.

Theoretically, you can also unlocked, flash, relock the bootloader. But you will use your TA partition.

**Comment by giosal** [comment]
So, I will not lose DRM keys and otherwise screw up the phone?

**Comment by bonanza123** [comment]
You will lose DRM keys.

**Comment by giosal** [comment]
That's why I don't want to unlock it...

**Comment by bonanza123** [comment]
I am afraid I don't know an other option. Do you have a reference for flashing official ROMs without requring unlocking the bootloader?

**Comment by giosal** [comment]
I've seen it somewhere on XDA, but don't remember to be honest, where exactly.
Plus, I remember flashing my previous Sony phones without unlocking bootloadaer.
Could you explain what exactly losing DRM keys will cause?

**Comment by bonanza123** [comment]
See https://www.xda-developers.com/sony-xperia-android-pie-unlock-bootloader-drm-fix-camera/

**Comment by giosal** [comment]
Thanks for all the info, but I guess I'll just wait for new ROM to come out for my region...

**Comment by bonanza123** [comment]
@giosal did you check https://support.sonymobile.com/de/xperia-companion/ ?

**Comment by giosal** [comment]
@bonanza123 of course I have, but the problem is that there is no Pie for my region yet (Customized FR)

**Comment by ArnauFernandez** [comment]
Bump. Getting this same issue with XA2 (H3113), anyone got a solution?

**Comment by nebulade** [comment]
Same for me with XA2 H3113 is there anything we can help debug this? Thanks a lot
